tajo-algebra/src/main/java/org/apache/tajo/algebra/WindowSpec.java

```diff
@@ -175,14 +175,14 @@ public Object clone() throws CloneNotSupportedException {
   }
 
   public static class WindowStartBound implements Cloneable {
```

Comment:
You're right, @jihoonson.
We don't need to keep them separately.
I'll merge them into ```WindowBound```.